src/main/java/io/jenkins/plugins/sprp/YamlRootAction.java

```diff
@@ -0,0 +1,74 @@
+package io.jenkins.plugins.sprp;
+
+import hudson.Extension;
+import hudson.model.UnprotectedRootAction;
+import hudson.security.csrf.CrumbExclusion;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * @author Abhishek Gautam gautamabhishek46
```

Comment:
this author tag is only used here in your plugin - if you're not going to use it everywhere, than I think this shouldn't be here